### PR TITLE
fix(docs): update jiva migration docs with affinity changes

### PR DESCRIPTION
**Pull request by shubham14bajpai**
Signed-off-by: shubham <shubham14bajpai@gmail.com>

**What this PR does**:
Updates the jiva volume migration docs. Currently tested on ci images. Will update the PR status once newly tagged images are available.
**Which issue(s) this PR fixes**:
Fixes #<issue number>

**Special notes for your reviewer**:

**Checklist**
- [ ] PR messages has document related information
- [ ] Labelled this PR & related issue with `documentation` tag
- [ ] PR messages has breaking changes related information
- [ ] PR messages has upgrade related information
- [ ] Labelled this PR & related issue with `requires-upgrade` tag
- [ ] Tests updated

### CODE REVIEW

File: docs/migration.md [hunk 1]
```diff
@@ -293,19 +293,30 @@ Before migrating the volumes make sure the following prerequisites are taken car
  - The jiva-operator should be installed with version 2.7.0 or above. You can install the correct version of jiva-operator from [charts](https://github.com/openebs/charts/tree/gh-pages). Get the jiva-operator yaml within the correct versioned folder and install. 
  - **The application needs to be scaled down before migrating.** This is required as the old PVC and PV spec needs to be deleted at the end of migration.
 
- For this example lets say the original volume has PVC name `demo-vol-claim` and PV name `pvc-898d90cc-ec73-4868-9205-1a8ba141a5bb`.
+ For this example lets say the original volume has PVC name `demo-vol-claim` and PV name `pvc-feefe71b-d073-4f0d-a5c2-bc0b78812683`.
```
Comment (nsathyaseelan) [inline docs/migration.md:296]: Include a note volume provisioned in default namespace

File: docs/migration.md [hunk 2]
```diff
@@ -360,26 +384,45 @@ Before migrating the volumes make sure the following prerequisites are taken car
...
 
-7. SSH into each node. Remove the files from the new localpv hostpath and copy the files from old hostpath to the new localpv hostpath which we have noted down in step 5. The old hostpath would be the same for all replica deployments and will be like `/var/openebs/<pv-name>`, for example `/var/openebs/pvc-898d90cc-ec73-4868-9205-1a8ba141a5bb`
+8. SSH into each node. Remove the files from the new localpv hostpath and copy the files from old hostpath to the new localpv hostpath which we have noted down in step 6. The old hostpath would be the same for all replica deployments and will be like `/var/openebs/<pv-name>`, for example `/var/openebs/pvc-feefe71b-d073-4f0d-a5c2-bc0b78812683`
 
-8. Scale up the replica statefulset. Replace the volume name in the application and scale it up. Verify the data.
+9. Scale up the replica statefulset. Replace the volume name in the application and scale it up. Verify the data.
 
-9. Delete the old volume. Done!!
+10. Delete the old volume. Done!!
     ```sh
```
Comment (nsathyaseelan) [inline docs/migration.md:426]: Post this PVC deletion seeing older PV still in released state can we include step to clear this as well

**Review by nsathyaseelan — CHANGES_REQUESTED** [review]

**Review by nsathyaseelan — APPROVED** [review]
lgtm

File: docs/migration.md [hunk 3]
```diff
@@ -293,19 +293,30 @@ Before migrating the volumes make sure the following prerequisites are taken car
...
 
- For this example lets say the original volume has PVC name `demo-vol-claim` and PV name `pvc-898d90cc-ec73-4868-9205-1a8ba141a5bb`.
+ For this example lets say the original volume has PVC name `demo-vol-claim` in `default` namespace and PV name `pvc-feefe71b-d073-4f0d-a5c2-bc0b78812683`.
  
 ### Migration steps
 
-1. Scale down volume controller and all replica deployments. You can find these deployments using the command:
+1. Note down the nodes on which the replicas were scheduled originally 
+   ```
+    $ k get pods -o wide
```
Comment (prateekpandey14) [inline docs/migration.md:302]: we can use pv-name label just to differentiate with other pods on per volume bases , helpful for the user to understand the no of pods ?

**Review by prateekpandey14 — COMMENTED** [review]
minor comment otherwise looks good

**Review by prateekpandey14 — APPROVED** [review]

**Pull request merged**